htdocs/supplier_proposal/class/supplier_proposal.class.php

```diff
@@ -518,10 +518,6 @@ public function addline($desc, $pu_ht, $qty, $txtva, $txlocaltax1 = 0, $txlocalt
 				$txtva = preg_replace('/\s*\(.*\)/', '', $txtva); // Remove code into vatrate.
 			}
 
-			if (!empty($conf->multicurrency->enabled) && $pu_ht_devise > 0) {
```

Comment:
I think we should keep this, because if the 2 value are provided (pu and pu_ht_devise), we want to have the  pu_ht_devise used in priority for prices calculation. So we need to force pu to zero for calcul_price_total.
May be you can keep only the line added later
`$pu = $pu_ht = $tabprice[3] fields ?`

Reply:
@eldy Thank you review.
You're right. 
I already fix it please check it.

Thank you.